app/views/companies/_form.html.erb

```diff
@@ -11,6 +11,18 @@
     </div>
   <% end %>
 
+  <div class="field">
+    <%= f.label :name %><br>
+    <%= f.text_field :name %>
+  </div>
+  <div class="field">
+    <%= f.label :notes %><br>
+    <%= f.text_field :notes %>
+  </div>
+  <div class="field">
+    <%= f.label :salesperson %><br>
+    <%= f.text_area :salesperson %>
```

Comment:
Mike and I were discussing it earlier and we decided that we were using that as a placeholder as we weren't 100% sure of the table columns.